app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -28,6 +29,27 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     has_many :middleware_messagings, :foreign_key => :ems_id, :dependent => :destroy
     has_many :middleware_server_groups, :through => :middleware_domains
 
+    standalone_operation :shutdown, 'Shutdown'
+    standalone_operation :suspend, 'Suspend'
+    standalone_operation :resume, 'Resume'
+    standalone_operation :reload, 'Reload'
+    standalone_operation :stop, 'Shutdown', {}, :original_operation => :Stop
+    standalone_operation :restart, 'Shutdown', { :restart => true }, :original_operation => :Restart
+
+    domain_operation :start, 'Start'
+    domain_operation :stop, 'Stop'
+    domain_operation :restart, 'Restart'
+    domain_operation :kill, 'Kill'
+
+    group_operation :start, 'Start Servers'
+    group_operation :stop, 'Stop Servers'
+    group_operation :restart, 'Restart Servers'
+    group_operation :reload, 'Reload Servers'
+    group_operation :suspend, 'Suspend Servers'
+    group_operation :resume, 'Resume Servers'
+
+    generic_operation :create_jdr_report, 'JDR'
```

Comment:
You can remove this one. It's dead code.
JDR op is implemented in [middleware_diagnostic_report](https://github.com/ManageIQ/manageiq-providers-hawkular/blob/master/app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb) model.